Resources/Private/Templates/Collection/List.html

```diff
@@ -36,14 +36,16 @@ <h2>
                         </span>
                     </f:if>
                 </h2>
-                <div class="tx-dlf-collection-thumbnail">
-                    <f:if condition="{item.collection.thumbnail}">
+                <f:if condition="{item.collection.thumbnail}"><div class="tx-dlf-collection-thumbnail">
+                    <f:link.action action="show"
+                                   arguments="{collection : item.collection}"
+                                   pageUid="{pageUid}">
```

Comment:
Yes, this makes sense and should work.

Could you please reformat the template tags? The source file was wrong, too. But you could fix it easily. E.g. like the following (should be examined in an IDE/Editor):
```suggestion
                <f:if condition="{item.collection.thumbnail}">
                    <div class="tx-dlf-collection-thumbnail">
                        <f:link.action action="show"
                                       arguments="{collection : item.collection}"
                                       pageUid="{pageUid}">
```